cg/apps/crunchy/crunchy.py

```diff
@@ -137,24 +135,24 @@ def is_fastq_compression_done(self, compression_obj: CompressionData) -> bool:
             return False
         LOG.info("SPRING metadata file found")
 
-        spring_metadata = self.get_spring_metadata(compression_obj.spring_metadata_path)
+        spring_metadata: CrunchyMetadata = get_spring_metadata(compression_obj.spring_metadata_path)
```

Comment:
Why have "spring" metadata on one side and "Crunchy" metadata on the other?

Reply:
fixed!